doc/latest_news.html

```diff
@@ -4,6 +4,7 @@
       December 22nd 2015: HoloViews 1.4.1 released and now available on 
       <a href="https://pypi.python.org/pypi/holoviews/">PyPI</a> and
       <a href="https://anaconda.org/anaconda/holoviews">Anaconda</a>.
+      Now includes full support for the <a href="http://bokeh.pydata.org">Bokeh</a> plotting library.
```

Comment:
Ok, I'll remove "full".  Looking at it in context, I decided that "extensive" might be a good replacement -- I do want to convey that this is not just a couple of things that are supported, but that there's been reallly a lot of work involved and it's completely viable for use in projects.